### ISSUE TITLE
Warning of lagacy support being used in Paper 1.21.1

### ISSUE THREAD

**Issue by Tsoccerguy3** [Lncvrt/AlwaysNightVision#3]
WARN]: Legacy plugin AlwaysNightVision v1.2.0 does not specify an api-version.
not a big deal, I just added a api to quiet the WARNING

**Comment by Lncvrt** [comment]
Yeah I have a uncommitted fix for this on my pc. I know what the issue is but you can ignore it

**Comment by Lncvrt** [comment]
I'll close this when I commit it
